src/LATTE/fix_latte.cpp

```diff
@@ -37,8 +37,8 @@ using namespace FixConst;
 extern "C" {
   void latte(int *, int *, double *, int *, int *,
              double *, double *, double *, double *,
-             double *, double *, double *, int*,
-             double *, double *, double *, double *, bool *);
+             double *, double *, double *, int *,
+             double *, double *, double *, double *, int * , bool *);
```

Comment:
If you are changing ABI you need to update the `LATTE_ABIVERSION` in `latte_lib.f90` and the abi check below.
Plus, you need to update the version downloaded in `lib/latte/Install.py` and `cmake/CMakeLists.txt`.

Reply:
- Change LATTE_ABIVERSION to something like 20180622?
- Ok, I will make a new latte release (v1.2.0) and and update Install.py and CMakeLists.txt.

Reply:
I changed both Install.py and CMakeLists.txt. I also did a release of latte v1.2.0

Reply:
But`LATTE_ABIVERSION` in v1.2.0 is updated: https://github.com/lanl/LATTE/blob/v1.2.0/src/latte_lib.f90#L65

Reply:
@junghans I think that should read, that ABIVERSION was **not** updated. It points to february 7th, 2018. The ABIVERSION commit seems to have happened **after** the release was tagged.

Reply:
The abi check in line 58 below needs an update!

Reply:
@akohlmey yes, that was a typo! @cnegre now we just new another LATTE release with the new abi version in it ;-)